README.md

```diff
@@ -30,52 +30,53 @@ _Data is initially pulled from Stanford's system, but the user can update certai
 
 | Column Name | Type | Meaning | Notes |
 | - | - | - | - |
-| `SUNet` | string (primary key) | SUNet id | never shown to users, only used internally |
-| `first_name` * | string | first name |  |
-| `last_name` * | string| last name |  |
-| `email` * | string (email address) | user's email address | assumed initially to be SUNet@stanford.edu, but user can update |
+| `user_id` | string (primary key) | uuid | |
+| `dining_hall_id` | string (foreign key) | uuid - dining hall the user is currently at |
 | `photo_path` * | string (file path) | path to the user's profile photo | relative to `/static/images/profile-photos` |
-| `swipes` * | integer | number of swipes the user currently has | we calculate this, but user can manually update to correct incorrect guesses |
-| `isDonor` * | boolean | if the user is a donor (true) or recipient (false) _right now_ |  |
-| `lifetime_donated` | integer | the total number of swipes the user has ever donated |  |
-| `lifetime_received` | integer | the total number of swipes the user has ever received |  |
+| `swipes_remaining` * | integer | number of swipes the user currently has | we calculate this, but user can manually update to correct incorrect guesses |
+| `is_donor` * | boolean | if the user is a donor (true) or recipient (false) _right now_ |  |
+| `major` | string | required field |  |
+| `biography` | string | short "about me" section |  |
 
 
 ### `Pairing` Table
 | Column Name | Type | Meaning |
 | - | - | - |
-| `id` | integer (primary key) | used to identify pairing |
-| `person_1` | User ID | one user in the pairing |
-| `person_2` | User ID | other user in the pairing |
-| `location` | DiningHall ID | place where the pair is getting their meal |
-| `meal` | Meal ID | meal that the pair is getting |
```

Comment:
I'm feeling like we don't need to store where the users are meeting or what meal they're getting. But lmk your thought process.

Reply:
agreed that it's not necessary for user experience, I was thinking it could be helpful on our end for data exploration / visualizations for the write-ups. you're right, should be low priority to implement

Reply:
also im literally so dumb these should be labelled donor and recipient

Reply:
> I was thinking it could be helpful on our end for data exploration / visualizations for the write-ups

oooo interesting. not opposed, although I wonder if people would come back to the app to report this. I'd say a definite v2 feature, but I'm open to whatevz!

Reply:
i was thinking we would store it when they got matched, so wouldn't require anything extra from the users

Reply:
makes sense, will re-add this back in 👍